tagstudio/src/qt/widgets/thumb_renderer.py

```diff
@@ -37,6 +37,10 @@ class ThumbRenderer(QObject):
     # finished = Signal()
     updated = Signal(float, QPixmap, QSize, str)
     updated_ratio = Signal(float)
+
+    # Use as static variable.
+    settings = QSettings()
```

Comment:
dont use a static variable, see my previous comment: https://github.com/TagStudioDev/TagStudio/pull/154#discussion_r1596154613


Or is this some paradigm I am not aware of? Honest question, I'm not that familiar with PySide guidelines.